src/textual/widget.py

```diff
@@ -803,16 +803,12 @@ def watch_hover_style(
             self.highlight_link_id = hover_style.link_id
 
     def watch_scroll_x(self, old_value: float, new_value: float) -> None:
-        if self.show_horizontal_scrollbar:
-            self.horizontal_scrollbar.position = round(new_value)
```

Comment:
This like is required to change the scrollbar.

Reply:
I'm reading *"like"* as *"line"* there, and assuming it's in reference to updating the scrollbar position. If so, good catch. Restored.